src/diff-pluses-and-minuses/diff-pluses-and-minuses.js

```diff
@@ -13,7 +13,7 @@ const stripCharsFromLine = (line: Element) => {
     if (line.textContent === '+' || line.textContent === '-') {
         line.textContent = ' '
     } else {
-        line.innerHTML = line.innerHTML.slice(1)
+        line.innerHTML = ' ' + line.innerHTML.slice(1)
```

Comment:
to keep the space when adding it to the clipboard, so everything is aligned, margin-right above is there to push the text back next to + or - where it was.

Reply:
Can you elaborate a little bit on why this is needed or wanted?

As a user, when copying diff lines I wouldn't want any extra content added to them, and that includes spaces.

Reply:
bitbucket already add spaces on line that don't have +/- so when removing the whitespace from lines changed, it changes the indentation when pasting.
```
       "UpstreamPathTemplate": "/api/{version}/sites",
      "UpstreamHttpMethod": [ "GET" ], // line changed, missing whitespace in front
      "UpstreamHttpMethod": ["GET"], // line changed, missing whitespace in front
       "AuthenticationOptions": {
         "AuthenticationProviderKey": "Auth0",
         "AllowedScopes": []
       }
```
also, once someone create a pull request to properly copy/paste code having this change here might help them determine which whitespace index to remove from each lines to have the proper indentation.
```
"UpstreamPathTemplate": "/api/{version}/sites",
"UpstreamHttpMethod": [ "GET" ],
"UpstreamHttpMethod": ["GET"],
"AuthenticationOptions": {
    "AuthenticationProviderKey": "Auth0",
    "AllowedScopes": []
}
```

Reply:
Got it, thanks!